windows/d2-installer.nsi

```diff
@@ -377,6 +377,38 @@ Function DetectVSAndSDK
     ClearErrors
 FunctionEnd
 
+; Get path component of file name
```

Comment:
I just copied the code from the NSIS wiki, I didn't expected to find it in the library, too. Updated.